.github/workflows/daily_common.yml

```diff
@@ -96,4 +96,3 @@ jobs:
 
           export NIMFLAGS="${NIMFLAGS} --mm:${{ matrix.nim.memory_management }}"
           nimble test
-          nimble testintegration
```

Comment:
instead of copying 99% the same workflow for integration.yml, could additional input be added that will determine if `nimble test` should be run, or `nimble testintegration` ?